contract_api/mpe.py

```diff
@@ -0,0 +1,3 @@
+class Channel:
```

Comment:
What does this class do?

Reply:
This got corrupted. Updated with the working copy. Please review.